content/docs/how-we-work/git/git-pr.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Pull Request (PR)
+weight: 2
+description: A Git PR, or pull request, is a way to propose changes to the codebase. Post review your changes are merged. This process helps to ensure that changes are reviewed by someone else, which reduces the chances of any errors.
+---
+
+# Pull Request (PR)
+
+A Git PR, or pull request, is a way to propose changes to the codebase. Post review your changes are merged. This process helps to ensure that changes are reviewed by someone else, which reduces the chances of any errors.
+
+By utilizing Git commands, we can [create and submit a pull request](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project#:~:text=Creating%20a%20Pull%20Request) to the repository owner and manage the entire process of review, approval, and merging of changes. The process involves creating a fork of the repository, cloning the fork to a local computer, making desired changes to the code, and committing these changes back to the repository. By submitting a PR, users can demonstrate how they have added value or addressed issues in the codebase, show evidence of their contributions, and allow the repository owner to evaluate the proposed changes prior to incorporating them into the codebase.
+
+## PR failures/debugging/troubleshoot
+
+Github makes some checks against our PR to see if it is ready to be merged. Failing these checks is considered a failing of PR, or simply PR failed.
```

Comment:
I see this document doesn’t mention GitHub Actions at all. I think it is important to mention that.